packages/core/src/payment/storefront-vaulting-request-sender.ts

```diff
@@ -0,0 +1,37 @@
+import { RequestSender } from '@bigcommerce/request-sender';
+
+import {
+    HostedFormVaultingData,
+    HostedFormVaultingInstrumentForm,
+} from '../hosted-form/hosted-form-vaulting';
+
+export default class StorefrontVaultingRequestSender {
+    constructor(private _requestSender: RequestSender) {}
+
+    async submitPaymentMethod(
```

Comment:
Shall we call this `submitPaymentInstrument` since payment method means something else in the context of SDK.

Reply:
@animesh1987 yeah, no problem. I'll rename it.